src/net/slashie/serf/ui/oryxUI/GFXUserInterface.java

```diff
@@ -1161,6 +1166,9 @@ public void shutdown(){
 		si.cleanLayer(getItemsLayer());
 		si.commitLayer(getItemsLayer(), true);
 		
+		si.getComponentsPanel().removeAll();

```

Comment:
no, not memory, the Black Frame stayed otherwise, even with the game being shut down.